rero_ils/modules/items/dumpers.py

```diff
@@ -52,8 +53,16 @@ def dump(self, record, data):
             'call_numbers': record.call_numbers,
             'location_name': location.get('name'),
             'library_name': location.get_library().get('name'),
-            'enumerationAndChronology': record.get('enumerationAndChronology')
+            'enumerationAndChronology': record.get('enumerationAndChronology'),
+            'item_type': record.get('name')
```

Comment:
Should it be record.get('item_type')['name'] ? I suspect this will return 'None'.